synapse/handlers/account_data.py

```diff
@@ -302,6 +303,15 @@ async def remove_tag_from_room(self, user_id: str, room_id: str, tag: str) -> in
             )
             return response["max_stream_id"]
 
+    async def push_rules_for_user(self, user: UserID) -> Dict[str, Dict[str, list]]:
```

Comment:
Is it possible to say what the inner `list` contains here?

Reply:
Yep. FTR I did just copy and paste this method!

I think it is unfortunately just a `List[JsonDict]`? But I suppose that's better than nothing.

Reply:
> I suppose that's better than nothing.

agreed!